src/components/Archive/Archive.tsx

```diff
@@ -65,23 +75,30 @@ export const Archive: FC = () => {
   const eventList = eventListData?.data ?? []
 
   return (
-    <Stack gap={2.5}>
+    <Stack gap={1}>
       {eventListIsLoading && <Loading />}
+
       {eventList.map((event) => (
-        <Stack key={event.id} direction="row" justifyContent="space-between">
-          <Typography variant="h3" component="span">
-            {event.year + '. ročník '}
-            {event.season_code === 0 ? 'zimný' : 'letný'}
-            {' semester'}
-          </Typography>
-          <Stack gap={2} direction="row" alignItems="center">
-            <ResultsButton eventYear={event.year} eventSeason={event.season_code} />
-            <ProblemsButton eventYear={event.year} eventSeason={event.season_code} />
-            {event.publication_set.map((publication) => (
-              <PublicationButton key={publication.id} publication={publication} />
-            ))}
-          </Stack>
-        </Stack>
+        <Accordion key={event.id} disableGutters square={false} sx={{boxShadow: 'none', '&:before': {display: 'none'}}}>
```

Comment:
bolo by super hodit este rovnaky archiv aj pre `src/components/CompetitionPage/CompetitionPage.tsx`

Reply:
follow-up: #677

Reply:
a na to oddelenie borderu na tlacitku: #676